scripts/lib/build-utils.mjs

```diff
@@ -283,6 +300,9 @@ export default {
       tasks.push(() => this.buildNerdctlStub('windows'));
       tasks.push(() => this.buildNerdctlStub('linux'));
     }
+    for (const os of ['windows', 'linux', 'darwin']) {
+      tasks.push(() => this.buildRdCtl(os));
```

Comment:
Why do we need to build the `darwin` binary on Win32 (and vice versa)?

Reply:
1. Building an OpenAPI spec is on the radar, but not part of the absolute minimal shippable feature. If I can get that in for the next release, great. But certainly at some point we should do that, and use swagger to generate the client and server -- fits in with my thought that right now the server re-invents a lot of wheels. They're small, elementary wheels right now, but we should stop doing that as well.

2. I was thinking of windows/linux, but should narrow that down.